.github/workflows/spellcheck.yaml

```diff
@@ -0,0 +1,14 @@
+name: Checking for spelling errors
+
+on:
+  push:
+
+jobs:
+  spellcheck:
+    name: rojopolis/spellcheck
+    runs-on: ubuntu-20.04
+    steps:
+    - name: actions/checkout
+      uses: actions/checkout@v2
+    - name: rojopolis/spellcheck - acutally doing something
```

Comment:
Typo in `acutally`.

Reply:
lmao that was actually a joke to see if anyone would see the typo in the spellchecker. ahhh the irony